File: openapi/code_samples/python/api_v1_organizations_{org_id}_roles_{role_name}/delete.py
# Basic delete
scalekit_client.roles.delete_organization_role(
    org_id="org_123",
    role_name="org_role_admin"
)

# With reassignment
scalekit_client.roles.delete_organization_role(
    org_id="org_123",
    role_name="org_role_admin",
    reassign_role_name="org_role_member"
)

File: openapi/code_samples/go/api_v1_organizations_{org_id}_roles_{role_name}/delete.go
// Basic delete
err := scalekitClient.Role().DeleteOrganizationRole(ctx, "org_123", "org_role_admin")
if err != nil { /* handle err */ }

// With reassignment
err = scalekitClient.Role().DeleteOrganizationRole(ctx, "org_123", "org_role_admin", "org_role_member")